ubuntu-advantage.1

```diff
@@ -75,6 +75,48 @@ Possible values are: active, inactive, or n/a
 Show version of the Ubuntu Advantage package.
 
 
+.SH CONFIGURATION
+By default, ubuntu-advantage client configuration options are read from
```

Comment:
We typically say "Ubuntu Advantage" above, so let's stick with that

Reply:
done.